pkg/storage/unified/sql/backend.go

```diff
@@ -103,6 +106,12 @@ func (b *backend) initLocked(ctx context.Context) error {
 		return fmt.Errorf("no dialect for driver %q", driverName)
 	}
 
+	// Process any data manipulation migrations
+	err = b.runStartupDataMigrations(ctx)
```

Comment:
Will those run every single time Grafana starts ?

Reply:
Also -- we should be able to remove this after a few months :)